python3-bionic/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM socrata/base-bionic
+
+RUN apt-get clean && apt-get update
+
+# Add locale profiles and default to en_US.UTF-8
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+
+# install some basic goodies
+RUN apt-get purge -y python.* && apt-get -y install build-essential libssl-dev gfortran gcc g++ libbz2-dev python3 python3-pip && ln -s /usr/bin/python3 /usr/local/bin/python && ln -s /usr/bin/pip3 /usr/local/bin/pip && ln -s /usr/bin/pydoc3 /usr/local/bin/pydoc && ln -s /usr/bin/python3-config /usr/local/bin/python-config
```

Comment:
I wonder if it makes sense to keep installing python from source separately, despite the redundancy, to allow us to more easily continue to version it independently of the OS version as time moves on and the OS version doesn't?   I'm not a python expert, but I'm also not sure of if there might be fallout from intermingling system python and the "application" python?

Reply:
I think using the OS python will be OK.

The Dockerfile removes all installed versions of python so there is no system python:
`RUN apt-get purge -y python.*`

Python 3.6 is in maintenance mode: https://www.python.org/dev/peps/pep-0494/ The OS python is 3.6.7-1~18.04 and only bug fix and security releases follow, which I think Canonical will handle promptly.

Reply:
I'd anticipate that we will use more versions than just whats found in the apt repo.  There's at least one confirmed project that uses 3.7.3.  This version, and moreso with 3.8.0, has features that 3.6.8 does not.

Reply:
I only created bionic images for images that are in use in marathon (in staging). I'd prefer to keep the scope of EN-33791 limited to that.

Reply:
I only created bionic images for images that are in use in marathon (in staging). I'd prefer to keep the scope of EN-33791 limited to that.